src/core/init/lifecycle.js

```diff
@@ -5,6 +5,8 @@ export function initLifecycle(vm) {
     'init',
     'mounted',
     'beforeEach',
+    'beforeEmbed',
+    'afterEmbed',
     'afterEach',
     'doneEach',
     'ready',
```

Comment:
We should list the order of hooks, otherwise people have to find this in the code right here.

Reply:
The updates to the "Write a Plugin" page do a better job of describing when hooks are invoked. The updates aren't live yet, but they can be viewed on the `develop` branch [preview](https://docsify-preview.vercel.app/#/write-a-plugin).